pkg/apis/storage/fuzzer/fuzzer.go

```diff
@@ -96,6 +96,10 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 					storage.VolumeLifecyclePersistent,
 				}
 			}
+			if obj.Spec.SELinuxMount == nil {
```

Comment:
https://github.com/kubernetes/kubernetes/blob/e660a1702b4955402d629ce518d66b9de1fb1b09/staging/src/k8s.io/api/storage/v1/types.go#L416 + internal types.go should be updated with the new "+featureGate" tag: https://github.com/kubernetes/kubernetes/blob/a9008b502d32a5dbacf835adb040419bf31f5c04/pkg/apis/core/types.go#L3141

Reply:
Added to v1 and internal types.go

Reply:
and v1beta1